google/services/cloudfunctions/resource_cloudfunctions_function_test.go

```diff
@@ -825,6 +825,18 @@ resource "google_cloudfunctions_function" "function" {
 func testAccCloudFunctionsFunction_firestore(functionName string, bucketName string,
 	zipFilePath string) string {
 	return fmt.Sprintf(`
+data "google_project" "project" {}
+
+resource "google_firestore_database" "database" {
+  project     = data.google_project.project.project_id
+  name        = "(default)"
```

Comment:
could you use a custom database name rather than default, then reference this database from the function? Or does the function always use the default database?